dts/bindings/power/nxp,aon-wakeup-pin.yaml

```diff
@@ -0,0 +1,22 @@
+# Copyright 2024, NXP
+# SPDX-License-Identifier: Apache-2.0
+
+description: Some NXP SoC's have pins dedicated to generate a wakeup interrupt.
+
+compatible: "nxp,aon-wakeup-pin"
+
+include: base.yaml
+
+properties:
+  interrupts:
+    type: array
+    required: true
+    description: Interrupt to wakeup the SoC from lowe power modes.
+
+  wakeup-level:
+    type: int
+    enum:
+      - 0
+      - 1
+    default: 1
+    description: Wakeup level on the pin, default is wakeup on high level.
```

Comment:
We need to justify the default here- is this the reset value for the PMU wakeup level? That is usually the reasoning behind a default